tfhe/src/typed_api/integers/types/static.rs

```diff
@@ -0,0 +1,449 @@
+        },
+    }
+}
+
+static_int_type! {
+    {
+        num_bits: 256,
+        keychain_member: integer_key.uint256_key,
+        parameters: Radix {
+            block_parameters: crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2,
+            num_block: 128,
+            wopbs_block_parameters: crate::shortint::parameters::parameters_wopbs_message_carry::WOPBS_PARAM_MESSAGE_2_CARRY_2,
+        },
+    }
+}
+
+
+//impl FheEncrypt<(u128, u128)> for GenericInteger<FheUint256Parameters> {
+//    #[track_caller]
+//    fn encrypt(value: (u128, u128), key: &ClientKey) -> Self {
+//        let id = <FheUint256Parameters as IntegerParameter>::Id::default();
+//        let key = id.unwrapped_ref_key(key);
+//        let ciphertext = key.inner.encrypt(u64::from(value));
+//        Self::new(ciphertext, id)
+//    }
+//}
+//
+//impl FheDecrypt<u8> for FheUint8 {
+//    #[track_caller]
+//    fn decrypt(&self, key: &ClientKey) -> u8 {
+//        let id = <FheUint8Parameters as IntegerParameter>::Id::default();
+//        let key = id.unwrapped_ref_key(key);
+//        key.inner.decrypt(&self.ciphertext.borrow()) as u8
+//    }
+//}
+
+
+impl FheEncrypt<u8, ClientKey> for GenericInteger<FheUint8Parameters> {
+    #[track_caller]
+    fn encrypt(value: u8, key: &ClientKey) -> Self {
+        let id = <FheUint8Parameters as ParameterType>::Id::default();
+        let key = id.unwrapped_ref_key(key);
+        let ciphertext = key.inner.encrypt(u64::from(value));
+        Self::new(ciphertext, id)
+    }
+}
+
+impl FheDecrypt<u8> for FheUint8 {
+    #[track_caller]
+    fn decrypt(&self, key: &ClientKey) -> u8 {
+        let id = <FheUint8Parameters as ParameterType>::Id::default();
+        let key = id.unwrapped_ref_key(key);
+        key.inner.decrypt(&self.ciphertext.borrow()) as u8
+    }
+}
+
+impl FheEncrypt<u16, ClientKey> for FheUint16 {
+    #[track_caller]
+    fn encrypt(value: u16, key: &ClientKey) -> Self {
+        let id = <FheUint16Parameters as ParameterType>::Id::default();
+        let key = id.unwrapped_ref_key(key);
+        let ciphertext = key.inner.encrypt(u64::from(value));
+        Self::new(ciphertext, id)
+    }
+}
+
+impl FheDecrypt<u16> for FheUint16 {
+    #[track_caller]
+    fn decrypt(&self, key: &ClientKey) -> u16 {
+        let id = <FheUint16Parameters as ParameterType>::Id::default();
+        let key = id.unwrapped_ref_key(key);
+        key.inner.decrypt(&self.ciphertext.borrow()) as u16
+    }
+}
+
+
+// impl FheTryEncrypt<crate::integer::U256, ClientKey> for FheUint256 {
+//     type Error = crate::typed_api::errors::Error;
+//
+//     fn try_encrypt(value: U256, key: &ClientKey) -> Result<Self, Self::Error> {
+//         let id = <FheUint256Parameters as IntegerParameter>::Id::default();
+//         let key = id.ref_key(key);
+//         let ciphertext = key.inner.encrypt(u64::from(value));
+//         Self::new(ciphertext, id)
+//     }
+// }
```

Comment:
missing newline at the end of the file

formatting is broken in the CI for this PR it seems